config.go

```diff
@@ -15,12 +15,30 @@ const (
 	defaultMessageWithTicketTemplate = "{{.TicketNumber}}{{if .Scope}}({{.Scope}}){{end}}{{if .IsBreakingChange}}!{{end}}: <{{.Type}}> {{.Message}}"
 )
 
+type LoadConfigReturn struct {
+	MessageTemplate           string
+	MessageWithTicketTemplate string
+	Prefixes                  []huh.Option[string]
+	Coauthors                 []huh.Option[string]
+	Boards                    []huh.Option[string]
+	CommitTitleCharLimit      int
+	ShowIntro                 bool
+}
+
 // loadConfig loads the config file from the current directory or any parent
-func loadConfig(fs afero.Fs) ([]huh.Option[string], []huh.Option[string], []huh.Option[string], bool, int, string, string, error) {
+func loadConfig(fs afero.Fs) (LoadConfigReturn, error) {
 	filePath, err := config.FindConfigFile(fs)
 	if err != nil {
 		log.Debug("Error finding config file", "error", err)
-		return config.DefaultPrefixes, nil, nil, true, defaultCommitTitleCharLimit, defaultMessageTemplate, defaultMessageWithTicketTemplate, nil
+		return LoadConfigReturn{
+			defaultMessageTemplate,
+			defaultMessageWithTicketTemplate,
+			config.DefaultPrefixes,
+			nil,
+			nil,
+			defaultCommitTitleCharLimit,
+			true,
+		}, nil
```

Comment:
It works, but it has many caveats.
Among them, I would like to mention:
- readability: what does the second nil state for?
- maintenance: you cannot easily add a field or reorder them
- source of bug: imagine you set the prefix in authors field
- you remove the logic of default value provided by the typed field in the struct

I would recommend you this


```suggestion
		return LoadConfigReturn{
			MessageTemplate: defaultMessageTemplate,
			MessageWithTicketTemplate: defaultMessageWithTicketTemplate,
			Prefixes: config.DefaultPrefixes,
			CommitTitleCharLimit: defaultCommitTitleCharLimit,
			ShowIntro: true,
		}, nil
```

Reply:
Ah that makes sense, I wasn't aware of the default value logic with typed fields!